Adding deploy script + documentation

visibilityspots opened the pull request:
I wrote a deploy script which can be used on the client side to easily deploy the pre-commit hook

drwahl commented on deploy-git-hook:
```diff
@@ -0,0 +1,13 @@
+
+if [ -f .git ];
+then
+	LOCATION="$(cat .git | awk '{print $2}')"
```
instead of 'cat file | awk', you can read the file directly with awk by:

awk '{print $2}' < .git

This would be the more "bash-y" way of doing this.

razorsedge replied:
Or you can just let awk handle it: `awk '{print $2}' .git`

drwahl commented on deploy-git-hook:
```diff
@@ -0,0 +1,13 @@
+	LOCATION=".git/hooks/pre-commit"
+fi
+
+ln -s /home/Jan/Projecten/Github.com/puppet-git-hooks/pre-commit $LOCATION
```
This home directory is unique to the user's install.  I suspect "/home/Jan/Porjecten/Github.com/puppet-git-hooks" should be something like "$HOME" or maybe "$HOME/git".  Something that is not user-specific.

visibilityspots commented:
Changed the awk command, wasn't aware of that feature. Cool!

About the path indeed something I totally overlooked. I changed it in the deploy-git-hook script to the $HOME directory but their should be a more flexible way to do so..

drwahl commented:
Looks good to me. Thanks for your contribution!
